docs/source/download.rst

```diff
@@ -165,7 +164,6 @@ On Ubuntu, the following apt packages are required:
 - libblas-dev
 - liblapack-dev
 - libopenmpi-dev
-- libspatialindex-dev
```

Comment:
Right, I need to add that somewhere. The system provided package won't be suitable in the near future, as we will switch to the C API.